### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by nataliebechtold**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+.card-bookmark{
+  position: absolute;
+  right: 5px;
+  top: -10px;
+}
+
+.inner-card {
+
+}
```
Comment (Kheirah) [inline styles.css:35]: Don't leave empty rulesets. Either implement or remove them ;-)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          <p class="flexbox-badge">#flexbox</p>
+          <p class="css-badge">#css</p>
+        </div>
+      </div>
+    </div>
+    <div class="navbar">
+      <img src="images/001-home.png" alt="">
+      <img src="images/211-bookmark.png" alt="">
+      <img src="images/profile-1.png" alt="" class="profile">
+    </div>
```
Comment (Kheirah) [inline index.html:32]: See if you can find semantic markup to improve its readability for screen readers and SEO.

**Review by Kheirah — APPROVED** [review]
LGTM

**Review by florianblaschke — APPROVED** [review]
Nice :)

**Pull request merged**